templates/CRM/Event/Form/Registration/ThankYou.tpl

```diff
@@ -22,11 +22,9 @@
     {/if}
 
     {* Show link to Tell a Friend (CRM-2153) *}
-    {if $friendText}
-        <div id="tell-a-friend" class="crm-section tell_friend_link-section">
-            <a href="{$friendURL}" title="{$friendText|escape:'html'}" class="button"><span><i class="crm-i fa-chevron-right" aria-hidden="true"></i> {$friendText}</span></a>
-       </div><br /><br />
-    {/if}
+  {foreach from='extensionText' item='text'}
```

Comment:
I think it would be more clear if these variables are used `extensionHtml` and `html`, instead of `extensionText` and `text`. That makes it more clear that the variables should not be escaped, and that the lack of escaping is intentional.